layouts/shortcodes/pentest-report-requirements.html

```diff
@@ -1,3 +1,3 @@
-If you want a pentest report, you generally must set up a test of at least two (2) credits. 
-If you have a one (1) credit pentest, you'll still have access to the non-report
+If you want a pentest report, you generally must set up a test of at least four (4) credits. 
```

Comment:
As suggested by @TheInstaGrahame 

```suggestion
If you want a pentest report, you generally must set up a test of at least five (5) credits. 
```

Reply:
> As suggested by @TheInstaGrahame

New Credit definition in platform does not support incremental credits such as 5,6,7,8,.... Such support will be available for assurance release. Until then for NCD release the credit increments are 4,8,12,16,... increments of 4. If PenOps plans to provide a report for a 4 credit pentest by additionally deducting 1 credit through Admin interface then this makes sense.

Reply:
After discussion with @TheInstaGrahame and @yogipetkar 

```suggestion
If you want a pentest report, you generally must set up a test of at least eight (8) credits. 
```